jsonmerger.py

```diff
@@ -0,0 +1,25 @@
+﻿import json

```

Comment:
This is a good-looking script! One small thing, and I apologize I didn't bring it up before, but just write a docstring at the top of your script using triple-double quotation marks - something like this:
```python
"""Script for joining scraped web data with sales data into a dataframe"""
```